ipaserver/plugins/user.py

```diff
@@ -366,6 +368,13 @@ class user(baseuser):
             },
             'default_privileges': {'PassSync Service'},
         },
+        'System: Manage User Certificate Mappings': {
```

Comment:
IMHO the similar permission should be added to stageuser implementation, something like `System: Manage Stage User Certificate Mappings`